netutils/config/parser.py

```diff
@@ -597,6 +597,130 @@ class F5ConfigParser(BaseBraceConfigParser):
 
     multiline_delimiters = ['"']
 
+    def __init__(self, config):
+        """Create ConfigParser Object.
+
+        Args:
+            config (str): The config text to parse.
+        """
+        super().__init__(self._clean_config_f5(config))
+
+    def _clean_config_f5(self, config_text):  # pylint: disable=no-self-use
+        """Removes all configuration items with 'ltm rule'.
+
+        iRules are essentially impossible to parse with the lack of uniformity,
+        therefore, this method ensures they are not included in ``self.config``.
+
+        Args:
+            config_text (str): The entire config as a string.
+
+        Returns:
+            str: The sanitized config with all iRules (ltm rule) stanzas removed.
+        """
```

Comment:
Should this function have an example? I don't know that we've defined is private functions should have an example. I just noticed this one doesn't and the one below does.

Reply:
I'll leave it to Jeff to decide, most private methods don't have it, but it is at times helpful to understand what is expected.

Reply:
I don't think it makes sense.  In most cases I wouldn't expect someone to directly interact with a private method, by putting an example on it and having it in the docs might influence that.